packages/httpsnippet-client-api/test/__datasets__/alternate-server/har.json

```diff
@@ -5,12 +5,6 @@
         "request": {
           "method": "POST",
           "url": "http://dev.local/v2/create/1234",
-          "headers": [
```

Comment:
Removed this from this fixture because the [server variables spec](https://github.com/readmeio/oas-examples/blob/main/3.0/json/server-variables.json) in `@readme/oas-examples` doesn't have any auth and we have dedicated tests for asserting auth anyways.